Airline_Web_API/Airline_Web_API/Services/JwtService.cs

```diff
@@ -42,10 +42,10 @@ public string GenerateJwtToken(User user)
         private IEnumerable<Claim> CreateClaims(User user)
         {
             var userIdClaim = new Claim(JwtRegisteredClaimNames.Sub, user.Id.ToString());
-            var userEmailClaim = new Claim(JwtRegisteredClaimNames.Email, user.Email);
-            var userRoleClaim = new Claim("role", user.Role);
+            var userNameClaim = new Claim(ClaimTypes.Name, user.Id.ToString());
```

Comment:
Why do you put user ID into user Name claim?

Reply:
I've method in Startup file which checks if user is authorized and this methods requires this value
![image](https://user-images.githubusercontent.com/56936671/97559947-22673780-19ef-11eb-846c-82d287d7ace0.png)

Reply:
I mean you already have user ID in Sub claim, why have in the Name as well?

Reply:
I corrected it. I didn't know how to do it, BUT now I've got it
![image](https://user-images.githubusercontent.com/56936671/97582892-564e5700-1a07-11eb-8c22-f7570f877c17.png)